classes/Http/Controller/Admin/TalksController.php

```diff
@@ -36,23 +35,6 @@ public function __construct(TalkHandler $talkHandler, Twig_Environment $twig, Ur
         parent::__construct($twig, $urlGenerator);
     }
 
-    public function rateAction(Request $request): Response
-    {
-        try {
-            $this->validate($request, [
```

Comment:
It looks like this was the last place that used the validate functio in the BaseController. Maybe we can remove it from the BaseController now